privacyidea/lib/tokens/hotptoken.py

```diff
@@ -298,6 +298,11 @@ def get_init_detail(self, params=None, user=None):
                                                   "value": oath_url,
                                                   "img": create_img(oath_url)
                                                   }
+                except KeyError as ex:
+                    audit_entry = self.find_most_recent_audit_entry(action="POST /auth")
+                    log.error("{0!s}".format((traceback.format_exc())))
```

Comment:
Please make the traceback a debug level. Usually we are doing tracebacks in debug.